cmd/sovereignnode/chainSimulator/configs/configs.go

```diff
@@ -0,0 +1,85 @@
+package configs
+
+import (
+	"encoding/json"
+	"math/big"
+	"os"
+
+	"github.com/multiversx/mx-chain-core-go/core"
+
+	"github.com/multiversx/mx-chain-go/common/factory"
+	"github.com/multiversx/mx-chain-go/config"
+	"github.com/multiversx/mx-chain-go/genesis/data"
+	"github.com/multiversx/mx-chain-go/integrationTests/chainSimulator"
+	chainSimulatorConfigs "github.com/multiversx/mx-chain-go/node/chainSimulator/configs"
+	"github.com/multiversx/mx-chain-go/node/chainSimulator/dtos"
+)
+
+var initialStakedEgldPerNode = big.NewInt(0).Mul(chainSimulator.OneEGLD, big.NewInt(2500))
+var initialEgldPerNode = big.NewInt(0).Mul(chainSimulator.OneEGLD, big.NewInt(5000))
+var initialSupply = big.NewInt(0).Mul(chainSimulator.OneEGLD, big.NewInt(20000000)) // 20 million EGLD
+
+// GenerateSovereignGenesisFile will generate sovereign initial wallet keys
+func GenerateSovereignGenesisFile(args chainSimulatorConfigs.ArgsChainSimulatorConfigs, configs *config.Configs) (*dtos.InitialWalletKeys, error) {
+	addressConverter, err := factory.NewPubkeyConverter(configs.GeneralConfig.AddressPubkeyConverter)
+	if err != nil {
+		return nil, err
+	}
+
+	initialWalletKeys := &dtos.InitialWalletKeys{
+		BalanceWallets: make(map[uint32]*dtos.WalletKey),
+		StakeWallets:   make([]*dtos.WalletKey, 0),
+	}
+	addresses := make([]data.InitialAccount, 0)
+	numOfNodes := int((args.NumNodesWaitingListShard+args.MinNodesPerShard)*args.NumOfShards + args.NumNodesWaitingListMeta + args.MetaChainMinNodes)
```

Comment:
You don't need all of that;
numShards = 1
metaNodes = 0
You only need the first part